src/components/cards/CardItem.vue

```diff
@@ -54,6 +54,11 @@
 						</div>
 					</transition>
 				</div>
+				<div v-if="!editing && compactMode" class="right">
+					<transition name="zoom">
```

Comment:
Let's get rid of the transition since we don't have that anywhere in nextcloud for a menu and it is not working anyways ;)

Reply:
Then you can also get rid of the wrapping div here.